runtime/lua/vim/_editor.lua

```diff
@@ -68,6 +68,12 @@ vim.log = {
   },
 }
 
+local utfs = {
+  ['utf-8'] = 'utf-8',
```

Comment:
I would make all the values `true` and inline the unnecessary references to `utfs` below.